cmd/helm/upgrade.go

```diff
@@ -177,6 +181,10 @@ func (u *upgradeCmd) run() error {
 	if err != nil {
 		return err
 	}
+	rawSecrets, err := vals(u.secretFiles, u.secrets)
```

Comment:
Make sure to pass `secretFiles` and `secrets` to the `installCmd` above. Otherwise if a user does `helm upgrade --install`, they won't end up using the secrets